packages/manager/apps/web/client/app/hosting/dashboard/hosting.service.js

```diff
@@ -528,23 +528,41 @@ import union from 'lodash/union';
       }
 
       /**
-       * Get list of user accounts that have access to webserver logs
+       * Get list of own logs linked to a hosting
        * @param {string} serviceName
+       * @param {string|null} fqdn
        */
-      getUserLogs(serviceName) {
-        return this.OvhHttp.get(`/hosting/web/${serviceName}/userLogs`, {
+      getOwnLogs(serviceName, fqdn) {
+        return this.OvhHttp.get(`/hosting/web/${serviceName}/ownLogs`, {
           rootPath: 'apiv6',
+          params: {
+            fqdn,
+          },
         });
       }
```

Comment:
```suggestion
      getOwnLogs(serviceName,id) {
        return this.OvhHttp.get(`/hosting/web/${serviceName}/ownLogs`, {
          rootPath: 'apiv6',
          params: {
           id,
          },
        });
      }
```

Reply:
param is fqdn for this route : https://api.ovh.com/console/#/hosting/web/%7BserviceName%7D/ownLogs~GET